deployments/helm/onos-gui/values.yaml

```diff
@@ -0,0 +1,233 @@
+
+replicaCount: 1
+
+image:
+  repository: onosproject/onos-gui
+  tag: latest
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+store:
+  raftgroup: onos-gui
+  controller: atomix-controller.kube-system.svc.cluster.local:5679
+
+## Client service.
+service:
+  enabled: true
+  ## Service name is user-configurable for maximum service discovery flexibility.
+  name: onos-gui
+  type: ClusterIP
+
+ingress:
+  enabled: true
+
+ports:
+  web:
+    containerPort: 80
+    protocol: TCP
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+onosservices:
+  topo:
+    grpc: 5150
+    proxy: 8080
+  config:
+    grpc: 5150
+    proxy: 8081
+#  control:
+#    grpc: 5150
+#    proxy: 8082
+
+Envoy:
+  # Inspired by https://github.com/helm/charts/blob/master/stable/envoy/values.yaml
+  image:
+    repository: envoyproxy/envoy
+    tag: v1.11.1
+    pullPolicy: IfNotPresent
+
+  command:
+    - /usr/local/bin/envoy
+  args:
+    - -l
+    - $loglevel
+    - -c
+    - /config/envoy.yaml
+
+  ports:
+    admin:
+      containerPort: 9901
+      protocol: TCP
+
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+  priorityClassName: ""
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+    # podAntiAffinity:
+    #   preferredDuringSchedulingIgnoredDuringExecution:
+    #     - weight: 50
+    #       podAffinityTerm:
+    #         topologyKey: failure-domain.beta.kubernetes.io/zone
+    #         labelSelector:
+    #           matchLabels:
+    #             release: envoy
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     - weight: 40
+    #       topologyKey: "kubernetes.io/hostname"
+    #       labelSelector:
+  #         matchLabels:
+  #           release: envoy
+
+  ## ref: https://github.com/envoyproxy/envoy/pull/2896
+  podAnnotations: {}
+    # prometheus.io/scrape: "true"
+  # prometheus.io/path: "/stats/prometheus"
+  # prometheus.io/port: "9901"
+
+  podLabels: {}
+  # team: "developers"
+  # service: "envoy"
+
+  livenessProbe:
```

Comment:
liveness and readiness are already defined in the deployment, why are re-defined here ?

Reply:
These are the `liveness` and `readiness` for the `onos-envoy` container.

The ones on lines 40 and 44 of `deployment.yaml` are for the `onos-gui` container.